### PR TITLE
Style updates after black format

### LINKED ISSUE #1926: Follow up #1876
- [x] change flake limit to 88
- [ ] adapt strange formattings

### PR DESCRIPTION
**Pull request by CaselIT**
Fixes #1926

- change flake limit to 88

### CODE REVIEW

**Comment by CaselIT** [comment]
did we leave out something else in the black pr?

also feel free to push or note other style changes to the code

File: tests/test_utils.py [hunk 1]
```diff
@@ -211,7 +211,8 @@ def test_pack_query_params_several(self):
             {'apples and oranges': '🍏 & 🍊'},
             {'garbage': ['&', '&+&', 'a=1&b=2', 'c=4&'], 'one': '1'},
             {'&': '&amp;', '™': '&trade;', '&&&': ['&amp;', '&amp;', '&amp;']},
-            # NOTE(vytas): Would fail because of https://github.com/falconry/falcon/issues/1872
+            # NOTE(vytas): Would fail because of
+            # https://github.com/falconry/falcon/issues/1872
             # {'&': '%26', '&&': '%26', '&&&': ['%26', '%2', '%']},
```
Comment (vytas7) [inline tests/test_utils.py:216]: Hm, it seems this could possibly be uncommented now?

**Review by vytas7 — APPROVED** [review]
:+1:

File: tests/test_utils.py (hunk 1 above)
Comment (CaselIT) [reply tests/test_utils.py:216]: good catch!

maybe it makes sense to use another pr though? just to keep things clear?

Comment (vytas7) [reply tests/test_utils.py:216]: Fine by me

**Review by vytas7 — APPROVED** [review]

File: tests/test_utils.py (hunk 1 above)
Comment (CaselIT) [reply tests/test_utils.py:216]: I've added https://github.com/falconry/falcon/issues/1940

**Pull request merged**